fury/decorators.py

```diff
@@ -43,3 +48,131 @@ def doctest_skip_parser(func):
         new_lines.append(code)
     func.__doc__ = "\n".join(new_lines)
     return func
+
+
+def warn_on_args_to_kwargs(from_version="0.1.O", until_version="0.11.0"):
+    """Decorator to enforce keyword-only arguments.
+
+    This decorator enforces that all arguments after the first one are
+    keyword-only arguments. It also checks that all keyword arguments are
+
+    expected by the function.
+
+    Parameters:
+    -----------
+    from_version: str
+        The version of fury from which the function was supported.
+    until_version: str
+        The version of fury until which the function was supported.
+
+    Returns:
+    --------
+    decorator: function
+        Decorator function.
+    """
+
+    def decorator(func):
+        """Decorator
+
+        Parameters:
+        -----------
+        func: function
+            Function to be decorated.
+
+        Returns:
+        --------
+        wrapper: function
+            Decorated function.
+
+        Examples:
+        ---------
+        >>> @warn_on_args_to_kwargs()
```

Comment:
you need to add a line to import otherwise, it will not work

Reply:
you men to import the decorator `warn_on_args_to_kwargs` in the  doctest ?